web/.env.defaults

```diff
@@ -0,0 +1,15 @@
+# Copy this file to .env to make modifications
```

Comment:
There are two of these? Is that correct?

The other one is up a level

Reply:
Web packs from inside web folder. /web/.env is read for web.
App packs from root folder.